mathgenerator/mathgen.py

```diff
@@ -531,6 +531,15 @@ def commonFactorsFunc(maxVal=100):
     problem = f"Common Factors of {a} and {b} = "
     solution = arr
     return problem, solution
+
+def quadraticEquation(maxVal=100):
+	a = random.randint(1,maxVal)
+	c = random.randint(1,maxVal)
+	b = random.randint(4*a*c,4*maxVal*maxVal)
+	
+	problem = "Zeros of the Quadratic Equation {a}x^2+{b}x+{c}=0".format(a,b,c)
+	solution = str[(-b+sqrt(b^2-4*a*c))/2*a,(-b-sqrt(b^2-4*a*c))/2*a]
```

Comment:
This was causing some errors for me, specifically with the sqrt. Make sure you test your code.